### ISSUE TITLE
Increase Security

### ISSUE THREAD

**Issue by root-intruder** [vinz486/Fingerprints-reader-R503---R502-esp8266-mqtt-Home-Assistant#17]
Currently, it is very easy to Hack the system. Thus, it should not be used for security relevant stuff.

Things that need to be done to improve security:

1. Set a password for R503 -> Hackers can not unplug the sensor and add their own fingerprint
2. set and save module addr (device_addr) on setup -> Hackers can not unplug the sensor and plug their own preconfigured sensor.

**Comment by vinz486** [comment]
Yes, both suggestions are very useful. But I have no time to develop any enhancement. I will accept any pr.

Thank you.
